src/test/java/racingcar/CarsTest.java

```diff
@@ -0,0 +1,72 @@
+package racingcar;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Test;
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+
+@SuppressWarnings("NonAsciiCharacters")
+public class CarsTest {
+
+    @Test
+    void 같은_이름의_자동차_생성시_예외발생() {
+
+        List<Car> cars = Arrays.asList(new Car("happy"), new Car("good"), new Car("happy"));
+
+        assertThatThrownBy(() -> new Cars(cars))
+            .isInstanceOf(IllegalArgumentException.class);
+
+    }
+
+    @Test
+    void 다른_이름의_자동차_생성_성공() {
+
+        Cars cars = new Cars(Arrays.asList(new Car("happy"), new Car("good")));
+
+        assertThat(
+            cars.getCars()
+                .stream()
+                .map(Car::getName)
+                .collect(Collectors.toList()))
+            .containsExactly("happy", "good");
```

Comment:
```suggestion
        assertThat(cars.getCars())
                .extracting("name")
                .containsExactly("happy", "good");
```
요렇게 추출할 수 있는 기능이 있답니다 :)

Reply:
assertThat 이후에 올 수 있는 메서드가 굉장히 많았군요! 다른 상황에서도 사용할 수 있게 다른 메서드들도 정리해봐야겠어요. 감사합니다 :)